internal/website/content/howto/blob/_index.md

```diff
@@ -9,3 +9,246 @@ providers and accessing them via HTTP. These guides show how to work with
 blobs in the Go CDK.
 
 <!--more-->
+
+## Opening a Bucket {#opening}
+
+The first step in interacting with unstructured storage is connecting to your
+storage provider. Every storage provider is a little different, but the Go CDK
+lets you interact with all of them using the [`*blob.Bucket` type][].
+
+[`*blob.Bucket` type]: https://godoc.org/gocloud.dev/blob#Bucket
+
+The easiest way to open a blob is using [`blob.OpenBucket`][] and a URL
+pointing to the blob, making sure you ["blank import"][] the driver package to
+link it in. See [Concepts: URLs][] for more details. If you need
```

Comment:
We should add a link here to the "providers" section below. E.g., something like "See [here][] for a list of supported providers, and how to construct URLs for each of them."

Reply:
(otherwise people think they know what a "URL" is but don't know how to map it in this context).

Reply:
I added one below. I am also about to rewrite this section for the issue #2387 that started all this scrutiny, so it's best to postpone additional content changes until then.